src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -78,18 +78,20 @@ private static Direction movementHelper(String newDirection) {
     }
 
     private static void implementCommand(List<String> parsedCommandList, List<String> roomExits) {
-        Room playerCurrentRoom = player.getCurrentRoom();
         if (parsedCommandList.size() == 1) {
             implementCommandOneWord(parsedCommandList);
         } else if (parsedCommandList.size() == 2){
-            implementCommandTwoWords(parsedCommandList,roomExits);
+            implementCommandTwoWords(parsedCommandList, roomExits, null);
         } else {
             narrativePanel.appendTextArea("Invalid Action");
         }
     }
 
-    private static void implementCommandTwoWords(List<String> parsedCommandList, List<String> roomExits) {
-        Room playerCurrentRoom = player.getCurrentRoom();
+    static void implementCommandTwoWords(List<String> parsedCommandList, List<String> roomExits, Player playerDependency) {
```

Comment:
What was the need for playerDependency?

Reply:
Need to be able to inject a Player object for testing. In the normal game operation, player passed as null, and if it's null, it'll be assigned to the Player instance instantiated by the Game class on startup.